### PR TITLE
added isString

### PR DESCRIPTION
**Pull request by rubiin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2348,6 +2349,44 @@ Checks if string ends with the given target string.
...
+ 
+  _.isString(123);
+  // => false
+  
+  _.endsWith('abc', 'b', 2);
+  // => true
+
+  // Native
+  function isString(str){
+    if (str && typeof str.valueOf() === "string") {
```
Comment (ODudek) [inline README.md]: Please add test case :)

Comment (rubiin) [reply README.md]: ok sure

Comment (rubiin) [reply README.md]: @ODudek Added tests

Comment (rubiin) [reply README.md]: @ODudek also is it ok if i add two or three features in a single pr?

Comment (ODudek) [reply README.md]: I think you should do another PR for features :)

Comment (rubiin) [reply README.md]: i will be adding maybe 2-4 features on lang. Should i do all those in different prs or a single one?

Comment (ODudek) [reply README.md]: If you want to add native implementation of different methods you should create different prs ;)

**Comment by rubiin** [comment]
do i need to address anything else for  the merge? @ODudek

**Comment by stevemao** [comment]
What is pull bot doing here???

**Comment by rubiin** [comment]
dont know

**Comment by rubiin** [comment]
can anyone reopen the issue @stevemao @ODudek  as its not merged

**Comment by stevemao** [comment]
I can't. There's no new commits here.

**Comment by NamPNQ** [comment]
@rubiin Are you do any force push to your master branch?

**Pull request closed without merging**